subsys/bluetooth/host/Kconfig

```diff
@@ -398,12 +398,24 @@ config BT_AUTO_PHY_UPDATE
 	depends on BT_PHY_UPDATE
 	default y if !BT_USER_PHY_UPDATE
 	help
-	  Initiate PHY Update Procedure on connection establishment.
+	  Initiate PHY Update Procedure on connection establishment. This will attempt
+	  to update the connection to use 2M PHY, however it doesn't actually guarantee
+	  that this is what will be used in the end.
+
+	  By default, this option only operates on central role connections. To also cover
+	  peripheral role connections the BT_CONN_AUTO_PHY_PNRIPHERAL option must be enabled.
 
 	  Disable this if you want the PHY Update Procedure feature supported
 	  but want to rely on the remote device to initiate the procedure at its
 	  discretion or want to initiate manually.
 
+config BT_CONN_AUTO_PHY_ON_PERIPHERAL
```

Comment:
`BT_CONN_AUTO_PHY_ON_PERIPHERAL` is a bit weird name I think. PERIPHERAL already implies CONN, and `BT_AUTO_PHY` would follow the naming of `BT_AUTO_PHY_UPDATE`.

Maybe something like `BT_AUTO_PHY_UPDATE_ON_PERIPHERAL` or `BT_PERIPHERAL_AUTO_PHY_UPDATE`?